### PR TITLE
Remove old view JS (bug 1134320)

### PR DESCRIPTION
**Pull request by muffinresearch**

### CODE REVIEW

File: webpay/base/templates/base.html [hunk 1]
```diff
@@ -84,10 +84,5 @@ <h2 class="heading"></h2>
       <div class="throbber"></div>
       <span class="txt"></span>
     </div>
-
-    {% block site_js %}
-      <script src="{{ settings.BROWSERID_JS_URL }}"></script>
-      {{ js('pay') }}
-    {% endblock %}
```
Comment (kumar303) [inline webpay/base/templates/base.html]: Can we just delete this whole template and anything that loads a media file? Or is that a rabbit hole for another bug?

**Comment by kumar303** [comment]
This is long overdue! Thanks.

r+wc

**Comment by kumar303** [comment]
Also, we may have a bug number for this but I can't find it so maybe not.

File: webpay/base/templates/base.html (hunk 1 above)
Comment (muffinresearch) [inline webpay/base/templates/base.html]: That'll probably break tests so I thought we should leave that. It'll be easier to see any regression doing this piece by piece.

**Pull request merged**